python/cugraph/cugraph/tests/sampling/test_node2vec.py

```diff
@@ -165,8 +163,22 @@ def test_node2vec(
     )
     num_verts = G.number_of_vertices()
     k = random.randint(6, 12)
-    start_vertices = cudf.Series(random.sample(range(num_verts), k), dtype="int32")
+    start_vertices = cudf.Series(
+        random.sample(range(num_verts), k), dtype=start_vertices_type
+    )
     max_depth = 5
+
+    if start_vertices_type == "int64":
+        warning_msg = (
+            "Node2vec requires 'start_vertices' to match the graph's "
+            "'vertex' type. input graph's vertex type is: int32 and "
+            "got 'start_vertices' of type: int64."
```

Comment:
Is it assumed that input graph's vertex type is: int32?  Shouldn't it be read from graph?
Or you are assuming that as the graph is small, the vertex type will always be int32?

Reply:
Yes [read_csv_file](https://github.com/rapidsai/cugraph/blob/branch-24.04/python/cugraph/cugraph/testing/utils.py#L225) creates a cudf DataFrame where the vertices are int32